examples/chip-tool/commands/clusters/ModelCommand.cpp

```diff
@@ -41,11 +41,13 @@ CHIP_ERROR ModelCommand::Run(PersistentStorage & storage, NodeId localId, NodeId
 {
     CHIP_ERROR err = CHIP_NO_ERROR;
 
+    InitInteractionModelDelegate();
```

Comment:
This should return a CHIP_ERROR (for memory allocation errors) and be checked for it.

since this really does an 'allocate if not yet allocated' a better name may be `EnsureIMDelegateAllocated`.